app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorHeader.vue

```diff
@@ -197,6 +220,14 @@ export default {
     closeQueriesModal() {
       this.isQueriesModalActive = false;
     },
+    resetQuery() {
+      this.$refs.savingDialog.disableSavingPrompt();
```

Comment:
Update the SavingDialog state via props, since such function is only triggered here.